src/app/file-browser/components/file-viewer/file-viewer.component.html

```diff
@@ -29,12 +29,29 @@
         *ngIf="isQueued(i)"
       ></div>
       <div *ngIf="currentRecord === record" class="full" [ngSwitch]="">
+        <pr-zooming-image-viewer
+          [item]="currentRecord"
+          (disableSwipe)="toggleSwipe($event)"
+          (isFullScreen)="toggleFullscreen($event)"
+          *ngIf="
+            showThumbnail &&
+            isZoomableImage &&
+            !isDocument &&
+            !isVideo &&
+            !isAudio
+          "
+        ></pr-zooming-image-viewer>
         <pr-thumbnail
           (disableSwipe)="toggleSwipe($event)"
           (isFullScreen)="toggleFullscreen($event)"
           [item]="currentRecord"
-          *ngIf="showThumbnail && !isDocument && !isVideo && !isAudio"
-          [hideResizableImage]="false"
+          *ngIf="
+            showThumbnail &&
+            !isZoomableImage &&
+            !isDocument &&
+            !isVideo &&
+            !isAudio
+          "
         ></pr-thumbnail>
```

Comment:
In theory I think we could get rid of this `pr-thumbnail` element completely, but I think we still want to use this for things like ZIP files and other archive file types.